C++/C++.sublime-syntax

```diff
@@ -24,6 +24,8 @@ file_extensions:
 first_line_match: |-
   (?xi:
     ^ \s* // .*? -\*- .*? \b(c\+\+|cpp\b) .*? -\*-  # editorconfig
+  | ^ \s* \# \s* include \s* <
+  | ^ \s* \# \s* include \s* "
```

Comment:
```suggestion
  | ^ \s* \# \s* include \s* [<"]
```

Reply:
makes sense. I originally wanted something like a complete match with filepath and closing bracket/quote, but quickly realized it would be an overkill.